src/components/uploader/css/Uploader.css

```diff
@@ -485,6 +485,10 @@ html>body .fl-uploader span.fl-uploader-browse {
     display:block;
 }
 
+.fl-uploader-basic-note {
+    font-weight: bold;
+}
+
```

Comment:
If the styling is only going to make the text bold, maybe use a `<strong>` element around the text instead.